### ISSUE TITLE
damage ticks

### ISSUE THREAD

**Issue by serumanooo** [diamond-rip/Eden#478]
So I looked for this. Looks like some spigot have knockback settings but do not allow to modify damage ticks. 

You can modify the damage ticks in kit menu if the damage ticks option isn't available in your spigot forks.

_Originally posted by @GoodestEnglish in https://github.com/diamond-rip/Eden/issues/387#issuecomment-1646682839_

**Comment by GoodestEnglish** [comment]
Have you change this variable?
![image](https://github.com/diamond-rip/Eden/assets/45838505/2d889f42-6000-4a17-b2eb-83466ba73025)

**Comment by serumanooo** [comment]
> Have you change this variable? ![image](https://user-images.githubusercontent.com/45838505/255353624-2d889f42-6000-4a17-b2eb-83466ba73025.png)

yes, it does not work for me, i tried everything to fix this, now im checking every single plugin configuration i have to see if i can make this work

**Comment by GoodestEnglish** [comment]
I've tested windspigot personally and it works. Most likely some misconfiguration on your side.

**Comment by serumanooo** [comment]
> I've tested windspigot personally and it works. Most likely some misconfiguration on your side.

another problem i have is invisible tnts, and i cant find the reason

**Comment by Antoniova1** [comment]
> He probado personalmente windspigot y funciona. Lo más probable es que haya alguna configuración incorrecta de su lado.



> > ¿Has cambiado esta variable?![imagen](https://user-images.githubusercontent.com/45838505/255353624-2d889f42-6000-4a17-b2eb-83466ba73025.png)
> 
> sí, no funciona para mí, intenté todo para arreglar esto, ahora estoy revisando cada configuración de complemento que tengo para ver si puedo hacer que esto funcione

have you solved it?  It doesn't work for me either for the combo kit

**Comment by serumanooo** [comment]
> > He probado personalmente windspigot y funciona. Lo más probable es que haya alguna configuración incorrecta de su lado.
> 
> > > ¿Has cambiado esta variable?![imagen](https://user-images.githubusercontent.com/45838505/255353624-2d889f42-6000-4a17-b2eb-83466ba73025.png)
> > 
> > 
> > sí, no funciona para mí, intenté todo para arreglar esto, ahora estoy revisando cada configuración de complemento que tengo para ver si puedo hacer que esto funcione
> 
> have you solved it? It doesn't work for me either for the combo kit

no, also windspigot is crashing my server every 15 minutes so i have to just stop using it

**Comment by Antoniova1** [comment]
> > > He probado personalmente windspigot y funciona. Lo más probable es que haya alguna configuración incorrecta de su lado.
> > 
> > 
> > > > ¿Has cambiado esta variable?![imagen](https://user-images.githubusercontent.com/45838505/255353624-2d889f42-6000-4a17-b2eb-83466ba73025.png)
> > > 
> > > 
> > > sí, no funciona para mí, intenté todo para arreglar esto, ahora estoy revisando cada configuración de complemento que tengo para ver si puedo hacer que esto funcione
> > 
> > 
> > have you solved it? It doesn't work for me either for the combo kit
> 
> no, also windspigot is crashing my server every 15 minutes so i have to just stop using it

that's weird if you want you can try lightspigot lite now i'm using it and it works fine for combo kit

**Comment by serumanooo** [comment]
> > > > He probado personalmente windspigot y funciona. Lo más probable es que haya alguna configuración incorrecta de su lado.
> > > 
> > > 
> > > > > ¿Has cambiado esta variable?![imagen](https://user-images.githubusercontent.com/45838505/255353624-2d889f42-6000-4a17-b2eb-83466ba73025.png)
> > > > 
> > > > 
> > > > sí, no funciona para mí, intenté todo para arreglar esto, ahora estoy revisando cada configuración de complemento que tengo para ver si puedo hacer que esto funcione
> > > 
> > > 
> > > have you solved it? It doesn't work for me either for the combo kit
> > 
> > 
> > no, also windspigot is crashing my server every 15 minutes so i have to just stop using it
> 
> that's weird if you want you can try lightspigot lite now i'm using it and it works fine for combo kit

idk, i already bought a spigot before and it was bad, im trying goatspigot
